students/BrandonHenson/Lesson3/strformat_lab.py

```diff
@@ -0,0 +1,66 @@
+# Brandon Henson
+# Lesson 3
+# StrFormat
+# 4/6/18
+
+
+def task_one():
+    print("__________Task One__________")
+    # Write a format string that will take the following four element tuple:
+    tuple1 = (2, 123.4567, 10000, 12345.67)
+    # and produce
+    formatted = 'file_{:0>4d} : {:3.2f} , {:.2e} , {:03.2e}'.format(*tuple1)
+    # 'file_002 :   123.46, 1.00e+04, 1.23e+04'
+    print(formatted)
+
+
+def task_two():
+    # Using your results from Task One, repeat the exercise,
+    # but this time using an alternate type of format string
+    # (hint: think about alternative ways to use .format() (keywords anyone?)
+    # and also consider f-strings
+    # if you’ve not used them already)
+    print("__________Task Two__________")
+    tuple_results = ('file_0002', 123.46, 1.00e+04, 1.23e+04)
+    print(f'{tuple_results[0]:s} :  {tuple_results[1]:.2f}, \
+    {tuple_results[2]:.2e}\, {tuple_results[3]:.2e}')
+
+
+def task_three(tuple1):
+    print("__________Task Three__________")
+    # Rewrite to take an arbitrary number of values.
+    # "the 3 numbers are: {:d}, {:d}, {:d}".format(1,2,3)
+    seq = []
+    for i in tuple1:
+        seq.append(i)
+    l = len(seq)
```

Comment:
Same comment as above on variable names, instead of assigning to `l`, I'd just use the call to `len(seq)` in the code below.